src/Workspaces/Core/Portable/Formatting/FormattingOptions.cs

```diff
@@ -65,6 +65,19 @@ private static Option<T> CreateOption<T>(OptionGroup group, string name, T defau
             defaultValue: false,
             storageLocations: EditorConfigStorageLocation.ForBoolOption("insert_final_newline"));
 
+        /// <summary>
+        /// Default value of 120 was picked based on the amount of code in a github.com diff at 1080p.
```

Comment:
when I measured the github diff width, I found it to be 130 (I don't mind a refactoring offered to wrap a little sooner though... maybe I'll be adding a new parameter right now and I want to wrap in advance)

Reply:
right.  i'm not intending this to be exactly the width of github.  but something reasonable that feels decent for large swaths of c# code, and also fits one of the largest community repos out there.  120 felt natural given it being just 1.5x of 80, another very common coding column (though hugely cramped in the .net space).